server/services/payment-service.js

```diff
@@ -1,5 +1,13 @@
 const paymentRepository = require('../repository/payment-repository')
 
+function paymentMapper (payment) {
+  return {
+    claimId: payment.claimId,
+    paymentAmount: Number.parseFloat(payment.value).toFixed(2),
```

Comment:
There's a lot of weird edge cases where rounding doesn't work as you'd expect in javascript. Not suggesting any changes here, but for a real payment system we'd need tests to ensure all the rounding work as we expect.